vhpc_toolkit/config_objs.py

```diff
@@ -1167,6 +1170,7 @@ def create_dvs(self, host_vmnics, dvs_name, num_uplinks=4):
                                 physical adapters.
             dvs_name (str): The name of the DVS to be created
             num_uplinks (int): Number of active uplinks
+            max_mtu: Max MTU for DVS
```

Comment:
same as above

Reply:
rename max_mtu to mtu, and also change the comment.